nbs/tutorials/methods.ipynb

```diff
@@ -11,10 +11,43 @@
    "cell_type": "markdown",
```

Comment:
<p>This won't work. You need to call <code>cchvae.train(...)</code> first.</p><p> </p><p>Please try to run them first. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/jax-relax/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Initially I didnt add train for similarity with prewritten documentation but have added it now.